src/plugins/GUITestBase/src/GTUtilsDashboard.cpp

```diff
@@ -146,6 +207,45 @@ bool GTUtilsDashboard::hasNotifications(HI::GUITestOpStatus &os) {
     return notificationsWidget->isVisible();
 }
 
+#define GT_METHOD_NAME "getNotifications"
+QList<GTUtilsDashboard::Notification> GTUtilsDashboard::getNotifications(GUITestOpStatus &os) {
+    using Type = Notification::Type;
+
+    const QString notificationsWidgetName = QStringLiteral("NotificationsDashboardWidget");
```

Comment:
Same here. Do not use string literals without special reason. Use QString notificationWidgetName = "";